src/common/AstroReactTable/AstroReactTable.tsx

```diff
@@ -33,17 +32,20 @@ export const AstroReactTable = ({
         {table.getFlatHeaders().map(({ id, column, getContext }: any) => (
           <div
             key={id}
-            className={classNames('Astro-react-table__col', {
-              'Astro-react-table__sorted': isSortable && !!column.getIsSorted(),
-              'Astro-react-table__sortable': isSortable,
-            })}
+            className={
+              isSortable && !!column.getIsSorted()
+                ? 'Astro-react-table__col Astro-react-table__sorted'
+                : 'Astro-react-table__col' || isSortable
+                ? 'Astro-react-table__col Astro-react-table__sortable'
+                : 'Astro-react-table__col'
+            }
```

Comment:
You can use a template literal so you don't have to repeat the `Astro-react-table__col` class in your ternaries.

```suggestion
            className={`Astro-react-table__col ${
              isSortable && !!column.getIsSorted()
                ? 'Astro-react-table__sorted'
                : isSortable
                ? 'Astro-react-table__sortable'
                : ''
            }`}

```